annileen/engine/mesh.ixx

```diff
@@ -76,20 +76,19 @@ namespace annileen
 		}
 	}
 
-	Mesh::Mesh()
-	{
-	}
-
 	Mesh::~Mesh()
 	{
 		unload();
+
+		// TODO: remove
+		std::cout << "Mesh destroyed." << std::endl;
 	}
 
 	MeshGroup::~MeshGroup()
 	{
-		for (auto& m : m_Meshes)
-		{
-			delete m;
-		}
+		m_Meshes.clear();
+
+		// TODO: remove
+		std::cout << "MeshGroup destroyed." << std::endl;
```

Comment:
hehe here too. also, why not `ANNI_LOG` instead of `std::cout`?

Reply:
I'll delete later. Can be merged. It was just for debug, but could use anni_log.